private/bufpkg/bufpackageversion/bufpackageversion.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020-2023 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bufpackageversion
```

Comment:
Why is this not in `bufcli`? This is where we've put all the rest of this CLI-specific stuff. Even moreso, `private/buf` is meant for things only used by the CLI, wheres `bufpkg` is shared across organization repositories, but since this package should be deleted it doesn't matter.